MediaSession.cpp

```diff
@@ -261,7 +261,11 @@ const char *MediaKeySession::GetKeySystem(void) const {
 
 DRM_RESULT DRM_CALL MediaKeySession::_PolicyCallback(
     const DRM_VOID *f_pvOutputLevelsData, 
-    DRM_POLICY_CALLBACK_TYPE f_dwCallbackType, 
+    DRM_POLICY_CALLBACK_TYPE f_dwCallbackType,
+#ifdef OPTEE
```

Comment:
I checked a unmodified v3.3 and v4.0 MS porting kit, and the function prototype for both is

    static DRM_RESULT DRM_CALL _PolicyCallback(
        __in     const DRM_VOID *f_pvOutputLevelsData,
        __in     DRM_POLICY_CALLBACK_TYPE f_dwCallbackType,
        __in_opt const DRM_KID *f_pKID,
        __in_opt const DRM_LID *f_pLID,
        __in     const DRM_VOID *f_pv );

So I don't think this is a OPTEE specific change, just a PR_3.3 change like the others.

Reply:
@petegriffin Actually Playready 3.3 Documentation reported it other way, that why moved under OPTEE thinking it's Linaro's OPTEE changes on top of 3.3.